src/lib.rs

```diff
@@ -15,7 +15,13 @@
 //!  * hash_pubkey_to_point: `BLS12381G2_XMD:SHA-256_SSWU_RO_` with the ASCII-encoded domain separation tag `BLS_POP_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_`
 //!
 //! While the Domain Separation Tag (DST) isn't hardcoded in this crate, we are hardcoding the choice of elliptic curve (BLS12-381), hash function (SHA-256), and variant (minimal-pubkey-size).
-use std::ops::{Add, AddAssign};
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Same question here: why is the `cfg_attr` needed? (this might be me not understanding how no-std support work!)

Reply:
this simply configures the crate & its dependencies for no-std if default features are turned off.

Reply:
Gotcha! Makes sense to me.